todo/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import TodoList from './components/Todo';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: [{
+          value: 'import redux library',
+          completed: false
```

Comment:
so keep in mind, that this info is going to be kept inside the redux store, not in this.state